packages/imodel-react-hooks/README.md

```diff
@@ -0,0 +1,227 @@
+    />
+  );
+};
+
+
+const MyApp = (props) => (
+  <FeatureOverrideReactProvider>
+    <A>
+      <B />
+      <C />
+    </A>
+  </FeatureOverrideReactProvider>
+);
+```
+
+`FeatureOverrideReactProvider` allows descendent components to set cascading feature overrides in viewports, and
+the overrides are executed in tree order of the components, so in the above example, overrides from `C` override `B`, which
+overrides `A`. The `overrider` property of `UseFeatureOverrides` is an analog for `FeatureOverrideProvider.addFeatureOverrides` which you
+would implement when adding your own vanilla JavaScript IModelJS FeatureOverrideProvider. This hook is useful for when you want multiple
+components to be able to control one `FeatureOverrideProvider` in cooperation, or when you don't want to manage notifying the viewport to
+refresh overrides yourself when you can do it on react state changes.
+
+There are no recipes for this hook yet, but there is room for one to be contributed.
+
+### `useFeatureOverrides(options: UseFeatureOverridesOpts, deps: any[]): void`
+
+| Option           | Type                                                                  | Note                                                                                                                                                                                                                         |
+| ---------------- | --------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| overrider        | `(overrides: FeatureSymbology.Overrides, viewport: Viewport) => void` | the code to run in the `FeatureOverrideProvider.addFeatureOverrides` function for this component                                                                                                                             |
+| completeOverride | `boolean | undefined`                                                 | whether to skip previous components in the component tree and go straight to this one, useful for performance savings when you're overriding everything and allowing earlier components to add overrides would be redundant. |
+
+### `FeatureOverrideReactProvider`
+
+| Property   | Type                                            | Description                                                                  | Default                           |
+| ---------- | ----------------------------------------------- | ---------------------------------------------------------------------------- | --------------------------------- |
+| viewFilter | `((viewport: Viewport) => boolean) | undefined` | A predicate function which filters which viewports to apply the overrides in | apply overrides in every viewport |
+
+## Maintainers
+
+- [Mike Belousov](mailto:Mike.Belousov@bentley.com)
```

Comment:
i would either remove this, or have it point to our github emails, leaning on removing this altogether since we have the codeowners file